.doctrine-project.json

```
@@ -7,7 +7,7 @@
     "versions": [
         {
             "name": "2.2",
-            "branchName": "master",
+            "branchName": "2.3.x",
```

Then the name should be `2.3`, right? 2.2 should get its own entry by now.

---

From what I saw in the past (last release) the changes to `.doctrine-project.json` were applied to `master` after the release. There indeed only branchName would need to change: https://github.com/doctrine/mongodb-odm/blob/4269eea27ffdbdc77c07161d59b8fbc06f1ce39d/.doctrine-project.json#L10

---

Ah right, a different workflow is used on the ODM repos. I'll do the changes here anyway so as to have only one PR and let the changes bubble up with a merge up, WDYT?

---

FWIW we can change this, I just had no clue what to do and followed what was done last time :)